src/Chapter2.hs

```diff
@@ -780,13 +788,13 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
--- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
+listElementsLessThan :: a -> [a] -> [a]
+listElementsLessThan x = filter (< x)
 
--- Can you eta-reduce this one???
-pairMul xs ys = zipWith (*) xs ys
+pairMul :: [a] -> [a] -> [a]
+pairMul = zipWith (*)
```

Comment:
The same comment about type signatures applies to these functions as well. They won't compile without constraints, so it's better to use some specific types at the moment